app/components/ingredient-list-edit.js

```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  ingredients: null,
+  allIngredientOptions: null,
+  // what options are in the dropdown should not contain ingredients 
+  // already selected
+  currentIngredientOptions: Ember.computed('allIngredientOptions.[]', 'ingredients.[]', {
+    get() {
+      let allIngredients = this.get('allIngredientOptions');
+      let ingredients = this.get('ingredients');
```

Comment:
What is *let*? 
Is it like declaring a variable? I never saw this one. Is it an ember built-in functionality for the components? 
I was expecting to see:
```javascript
var allIngredients;
allIngredients = this.get('allIngredientOptions');
```
Is it an equal replacement? Just wan to know. Never used it before.